main.py

```diff
@@ -3,5 +3,30 @@
 from django.db.models import QuerySet
 
 
+from db.models import Actor, Genre
+
+
 def main() -> QuerySet:
-    pass
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to use the correct name initially to avoid unnecessary updates.